doc/overview.rst

```diff
@@ -629,108 +639,81 @@ Tabular
+   -  You must set up AWS credentials. For more information, see `Configuring the AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config>`__.
    -  Your IAM role must allow connection to Amazon SageMaker and
       Amazon S3. For more information about IAM role permissions,
       see `Policies and permissions in IAM <https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html>`__.
 
-Built-in Components
-===================
-
-The following sections give information about the main built-in
-components and their function.
-
-Pre-trained models
-------------------
-
-SageMaker maintains a model zoo of over 300 models from popular open source model hubs, such as
-TensorFlow Hub, Pytorch Hub, and HuggingFace. You can use the SageMaker Python SDK to fine-tune
-a model on your own dataset or deploy it directly to a SageMaker endpoint for inference.
 
-Model artifacts are stored as tarballs in a S3 bucket. Each model is versioned and contains a
-unique ID which can be used to retrieve the model URI. The following information describes the
-``model_id`` and ``model_version`` needed to retrieve the URI.
+Deploy a Pre-Trained Model Directly to a SageMaker Endpoint
+============================================================
 
-.. container::
+You can deploy a built-in algorithm or pre-trained model to a SageMaker endpoint in just a few lines of code using the SageMaker Python SDK.
 
-   -  ``model_id``: A unique identifier for the JumpStart model.
-   -  ``model_version``: The version of the specifications for the
-      model. To use the latest version, enter ``"*"``. This is a
-      required parameter.
+First, find the model ID for the model of your choice in the :doc:`Built-in Algorithms with pre-trained Model Table<./doc_utils/pretrainedmodels>`.
 
-To retrieve a model, first select a ``model ID`` and ``version`` from
-the :doc:`available models <./doc_utils/pretrainedmodels>`.
+Low-code deployment with the JumpStartModel class
+-------------------------------------------------
 
+Using the model ID, define your model as a JumpStart model. Use the ``deploy`` method to automatically deploy your model for inference.
+In this example, we use the FLAN-T5 XL model from HuggingFace.
 .. code:: python
 
-   model_id, model_version = "huggingface-spc-bert-base-cased", "1.0.0"
-   scope = "training" # or "inference"
+    from sagemaker.jumpstart.model import JumpStartModel
 
-Then use those values to retrieve the model as follows.
+    model_id = "huggingface-text2text-flan-t5-xl"
+    my_model = JumpStartModel(model_id=model_id)
+    predictor = my_model.deploy()
+
+You can then run inference with the deployed model using the ``predict`` method.
 
 .. code:: python
 
-   from sagemaker import model_uris
+    question = "What is Southern California often abbreviated as?"
```

Comment:
can we use a more generic payload, like `hello world!`? I don't want to set the expectation that this is the kind of payload that *should* be given.

Reply:
Yep, can do -- will make both of these more generic

Reply:
Could you check with @khetan2 please?

Since this is a generative model, I feel like we should ask it a question.

Reply:
This payload was the example for another model, so I'm not sure it's appropriate here

Reply:
After thinking about this more and discussing with Ashish, FLAN-T5 XL is currently our #1 recommended model for question/answering, so this example prompt is fine here. GPT-J (the inference example model) can also be used for question/answering, but is more flexible for additional use cases. We can mention that GPT-J can be used for things like code generation, summarization, and more, but the q&a example will still be applicable. In general, customers can explore our documentation, the pretrained models table, and example notebooks to find appropriate use cases for each model.